seed/seedwriter/writer.go

```diff
@@ -466,9 +466,26 @@ func IsSytemDirectoryExistsError(err error) bool {
 	return ok
 }
 
+// correctedValidationSetAtSequence returns the corrected AtSequence for an
+// validation set. If any restrictions have been set in
+// in the manifest, then we must use the sequence and pinning status from
+// that instead of whats set in the model.
+func (w *Writer) correctedValidationSetAtSequence(vsm *asserts.ModelValidationSet) *asserts.AtSequence {
```

Comment:
corrected make it sounds like there's an error involved? maybe adjusted,final,finalized?

Reply:
I like the suggestions, I went with `final`